lms/djangoapps/grades/models.py

```diff
@@ -257,8 +257,14 @@ def save_grade(cls, **kwargs):
                     usage_key=usage_key,
                     defaults=kwargs,
                 )
+            log.info("persistent subsection grade successfully saved: {0}".format(unicode(grade)))
         except IntegrityError:
             cls.update_grade(user_id=user_id, usage_key=usage_key, **kwargs)
+            log.info("integrity error trying to save grade for user {0}, usage key {1}, defaults {2}".format(
```

Comment:
Although this is an expected error because of race conditions, can we make this a `warning` so we can keep an eye on them?
Also, can you log the exception stack trace?  `log.exception` will automatically add the stacktrace, but warning won't.